tests/unit/ui/components/TestExpandableCard.js

```diff
@@ -32,24 +44,17 @@ describe(__filename, () => {
   });
 
   it('toggles when clicked', () => {
-    // We have to mount for this test because of the usage of `setState`
-    // in this component.
-    const root = mount(<ExpandableCard i18n={fakeI18n()} />);
-    const card = () => root.find('.ExpandableCard');
-    const fakeEvent = createFakeEvent();
+    const { store } = dispatchClientMetadata();
 
-    // Clicking on the toggle should expand the card.
-    root.find('.ExpandableCard-ToggleLink').simulate('click', fakeEvent);
+    const root = render({ store });
 
-    sinon.assert.called(fakeEvent.preventDefault);
-    expect(card()).toHaveClassName('ExpandableCard--expanded');
+    // We want to simulate card's link onclick.
+    // TODO: look into why this isn't available with find
```

Comment:
The element with `onClick` is not part of the root which is why you can't get it with `root.find()`. It's part of the JSX which was passed to the `header` prop of `Card`. You'll need to get that prop and pass it to `shallow()`. That will give you a new shallow wrapper root that you can simulate `click` on.

I mentioned it in the *Testing JSX passed through properties* of [this article](https://hacks.mozilla.org/2018/04/testing-strategies-for-react-and-redux/) which goes into detail about how to do it with some example code.

Reply:
ahh ok yes! thank you